src/com/reason/esy/EsyProcess.java

```diff
@@ -1,104 +1,65 @@
 import com.intellij.openapi.vfs.VirtualFile;
-import com.intellij.util.ObjectUtils;
-import com.reason.*;
 import com.reason.Compiler;
-import com.reason.dune.DuneOutputListener;
+import com.reason.*;
 import com.reason.ide.ORProjectManager;
 import com.reason.ide.console.CliType;
+import org.jetbrains.annotations.Nls;
 import org.jetbrains.annotations.NotNull;
 import org.jetbrains.annotations.Nullable;
 
-import java.io.BufferedReader;
-import java.io.IOException;
-import java.io.InputStreamReader;
 import java.nio.file.FileSystem;
 import java.nio.file.FileSystems;
 import java.nio.file.Path;
-import java.util.Map;
 import java.util.Optional;
-import java.util.Set;
 import java.util.concurrent.atomic.AtomicBoolean;
 import java.util.function.Consumer;
+import java.util.function.Function;
 
 import static com.intellij.notification.NotificationType.ERROR;
-import static com.intellij.notification.NotificationType.WARNING;
-import static com.reason.dune.DuneConstants.DUNE_EXECUTABLE_NAME;
 import static com.reason.esy.EsyConstants.ESY_EXECUTABLE_NAME;
 import static com.reason.esy.EsyProcessException.esyNotFoundException;
 
 public class EsyProcess implements CompilerProcess {
 
-  private static final Runnable SHOW_DUNE_NOT_FOUND_NOTIFICATION =
-      () -> Notifications.Bus.notify(new ORNotification("Dune Missing", "Unable to find dune executable in esy PATH.", ERROR));
-
+  @Nls
   private static final Runnable SHOW_ESY_NOT_FOUND_NOTIFICATION =
-      () -> Notifications.Bus.notify(new ORNotification("Esy Missing", "Unable to find esy executable in system PATH.", ERROR));
+          () -> Notifications.Bus.notify(new ORNotification("Esy Missing",
+                  "Unable to find esy executable in system PATH.", ERROR));
 
-  private static final Runnable SHOW_ESY_PROJECT_ROOT_NOT_FOUND_NOTIFICATION =
-          () -> Notifications.Bus.notify(new ORNotification("Missing Project Configuration File",
-                  "Unable to find esy package.json. Has it been created?", ERROR));
+  @Nls
+  private static final Runnable SHOW_ESY_PROJECT_NOT_FOUND_NOTIFICATION =
+          () -> Notifications.Bus.notify(new ORNotification("Esy Project Not Found",
+                  "Unable to find esy project. Have you run esy yet?", ERROR));
 
+  @Nls
   private static final Consumer<Exception> SHOW_EXEC_EXCEPTION_NOTIFICATION =
-      (e) -> Notifications.Bus.notify(new ORNotification("Esy Exception", "Failed to execute esy command.\n" + e.getMessage(), ERROR));
+          (e) -> Notifications.Bus.notify(new ORNotification("Esy Exception",
+                  "Failed to execute esy command.\n" + e.getMessage(), ERROR));
 
-  private static final Consumer<Exception> SHOW_DUNE_EXCEPTION_NOTIFICATION =
-      (e) -> Notifications.Bus.notify(new ORNotification("Dune Exception", "Failed to execute dune command.\n" + e.getMessage(), ERROR));
-
-  private static final Log LOG = Log.create("esy");
+  private static final Log LOG = Log.create("compiler.esy");
```

Comment:
it is esy.compiler in antoher file

Reply:
copy-pasted. thanks for noticing this. renamed to "process.esy"